Source/Lib/WAV/WAV.cpp

```diff
@@ -136,23 +136,23 @@ bool riff::Parse(bool AcceptTruncated, bool FullCheck)
         {
             Name = Get_B4();
             Size = Get_L4();
-            if (Name == 0x52494646) // "RIFF"
+            if (Name == 0x52494646) // "WAV"

```

Comment:
"WAV" = 0x574156

Reply:
Bad find and replace, it is RIFF in the file.
I'll revert to if (Name == 0x52494646) // "RIFF"

Reply:
It’s the kind of incongruence that Travis & Co. overlooks ;-)